app/data/zsh-completion/_scrcpy

```diff
@@ -73,6 +73,8 @@ arguments=(
     '--record-orientation=[Set the record orientation]:orientation values:(0 90 180 270)'
     '--render-driver=[Request SDL to use the given render driver]:driver name:(direct3d opengl opengles2 opengles metal software)'
     '--require-audio=[Make scrcpy fail if audio is enabled but does not work]'
+    '--root[Launch the server as root]'
+    '--rotation=[Set the initial display rotation]:rotation values:(0 1 2 3)'
```

Comment:
Why are you adding rotation here?

Reply:
I'm assuming upstream ```dev``` had left it alone, but it's just a cosmetic change for those running ```scrcpy``` under ```zsh```. The file in question is just an auto-complete/help file associated with the program. I double-checked the file and there was no double mention of ```--rotate=```.

Reply:
That, and I also figured it'd be a good idea to include it until such time that the ```--rotate``` flag actually gets un-implemented in upstream ```dev```.